### DISCUSSION TITLE
Query regarding pixel of images in grid box

### DISCUSSION THREAD

**Discussion by Mangesh421** [cvat-ai/cvat#2595] [Q&A]
Hello All,
               I am new in CVAT . I want to understand how to calculate pixel with help of grid size in CVAT. As their is no provision for measuring pixels in CVAT. Can anyone suggest solution/way to find out pixels?

**Comment by azhavoro** [comment]
> I am new in CVAT . I want to understand how to calculate pixel with help of grid size in CVAT. As their is no provision for measuring pixels in CVAT. Can anyone suggest solution/way to find out pixels?

@Mangesh421  Hi, sorry for the late response, could you please describe what exact you want to measure, the H,W values of box or corner(s) position in pixel?
